recipes/grpcio/set_109_deployment_target.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index 67c36a9..7b1b709 100644
+--- a/setup.py
++++ b/setup.py
+@@ -193,7 +193,7 @@ if 'darwin' in sys.platform and PY3:
+   mac_target = sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET')
+   if mac_target and (pkg_resources.parse_version(mac_target) <
+                      pkg_resources.parse_version('10.7.0')):
+-    os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.7'
++    os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.9'
```

Comment:
Is this necessary? `MACOSX_DEPLOYMENT_TARGET` is 10.9 and the if condition is false